web/src/components/apps/AppDetailPage.tsx

```diff
@@ -97,7 +97,7 @@ function AppDetailPage(props: Props) {
     data: appsData,
     error: appsError,
     isError: appsIsError,
-    isLoading: appsIsLoading,
+    isFetching: appIsFetching,
```

Comment:
`isLoading` will always be false after the initial load. we want to use `isFetching` instead